pkg/operator/ceph/object/zone/controller.go

```diff
@@ -187,6 +188,14 @@ func (r *ReconcileObjectZone) reconcile(request reconcile.Request) (reconcile.Re
 		return reconcileResponse, err
 	}
 
+	// RECONCILE POOLS
+	logger.Info("reconciling object zone pools")
+	objContext := object.NewContext(r.context, r.clusterInfo, cephObjectZone.Name)
+	err = object.CreatePools(objContext, cephObjectZone.Spec.MetadataPool, cephObjectZone.Spec.DataPool)
```

Comment:
Should this be done inside `createCephZone()` a couple lines down? The pools are part of the zone CR.